example/package.json

```diff
@@ -21,7 +21,7 @@
     "deploy-cumulus": "cd cumulus-tf && terraform init && terraform apply",
     "deploy-data": "cd data-persistence-tf && terraform init && terraform apply",
     "int-test": "../node_modules/.bin/jasmine && npm run parallel-tests",
-    "package": "for x in lambdas/*; do cd $x && npm run package && cd -; done",
+    "package": "for x in lambdas/*; do (echo \"packaging $x\" && cd $x && test -e package.json && npm run package); done",
```

Comment:
So - to be clear - the intention here is that `package` still fails, but it does so without hanging (weirdly this situation *does* cause bootstrap failures for me going `feature-rds` -> `master`) and gives you some indication of what failed in the log in addition to the lerna spam?

Reply:
No actually package just doesn't do anything in a directory that doesn't have a package.json file.
so it will still run and package all of the lambdas in example/lambdas/*, but won't break if you have another directory under example/lambdas/whatever without a package.json file in it.

Reply:
Hmm, ok.    I was going to comment that this wouldn't fail, but if that's the intention.... 

I suppose in the case of example lambdas, if the package is expected to be there it would fail on terraform apply, but you could *in theory* mask a problem if a built artifact was present, but somehow the package.json got removed.          

I also just realized this is a *draft* PR, so I'll leave it be for now, but I'll be happy to have it in as cleaning my repo is a daily annoyance.

Reply:
How will this fail on TF apply? out of curiosity (since the integration tests aren't passing)

Reply:
It's out of draft, so let's fight 🦃 🥊

Reply:
it failing on TF apply wasn't a *criticism*, I just mean if a lambda resource that's supposed to be there doesn't build, the TF will fail if it *should* have built.

Reply:
Really the edge case I'd be concerned with is : 

Locally:
1) Prior build runs, adds 'stuff' to dist
2) Switch branch, branch no longer has package.json/code/whatever, and would fail 
3) Do local dev, deployment/other things work 
CI: 
4)  Oh no, integration tests fail. 

That said, the benefit is worth ^ being possible.

Reply:
I'm hoping this wouldn't affect the CI, I would expect them to pass since they get checked out fresh. I'm really just hoping someone else doesn't spend a couple of days trying to build locally without any information being provided when a random dir causes an infinite loop.